### PR TITLE
Iansui: 1.003 added

### LINKED ISSUE #7742: Add Iansui
**Font Project Git Repo URL:**
https://github.com/ButTaiwan/iansui

**Super short description of the Font Family:**

- Iansui is a Traditional Chinese font, which is divided from Klee One Project.
- Iansui complies with the font specifications of Taiwan's Ministry of Education and is suitable for education and school-age reading.
- Iansui supports Taiwanese, Hakka Chinese characters, and phonetic symbols of Taiwanese, Hakka, and aboriginal languages.

**Requirements:**

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [x] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [x] The entire font project is licensed under the [OFL](https://openfontlicense.org/open-font-license-official-text/)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/blob/main/data/results/txt/nice-names/GF_Latin_Core.txt)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I am the author of the font, or the author agreed to this submission
- [x] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**

image

Attach here a pic or a screenshot of the font. One image is enough, it can be a few letters, to give a quick overview.

### PR DESCRIPTION
**Pull request by aaronbell**
Taken from the upstream repo https://github.com/ButTaiwan/iansui at commit https://github.com/ButTaiwan/iansui/tree/41190e8094a9a5b47626c5d927daf0dcb226babf

New font project as part of the GF CJK Q4 onboarding effort. Resolves #7742. 

Some existing `FAILS` with FB that I do not believe are problematic at this time.
`com.google.fonts/check/glyph_coverage` - Font contains GF Core
`com.google.fonts/check/dotted_circle` - Font is primarily intended for CJK use
`com.google.fonts/check/case_mapping` - This is an upstream (Klee) issue that I don't think needs attending to for this project at this time.

### CODE REVIEW

**Comment by aaronbell** [comment]
BTW, for the `METADATA` file, it isn't clear to me the difference between a `"chinese-hongkong"` subset and `"chinese-traditional"` subset. I followed what was in Cactus Classical Serif, which is a similar category of font to this one, but figured it is worth asking.

**Comment by m4rc1e** [comment]
Images in articles must be stored within the article dir. See how Playwrite has done it, https://github.com/google/fonts/tree/main/ofl/playwriteno/article

**Comment by aaronbell** [comment]
@m4rc1e Thanks Marc! Done.

**Comment by emmamarichal** [comment]
Hi @aaronbell,

Fontbakery looks good to me!


I just have a question about Cyrillic, I can see that several glyphs have been drawn, but I don't suppose they'll do any good given the spacing?

Screenshot 2024-11-27 at 15 54 34

**Comment by aaronbell** [comment]
@emmamarichal I'm sorry, I must have missed your comment. This sort of situation is not uncommon with Japanese fonts as Cyrillic (and Greek) is expected to be included, but is usually designed as full width rather than proportional. As Iansui is based on a Japanese font, it has the same characteristic. 

GF has allowed in many fonts with similar characteristics.

**Review by emmamarichal — APPROVED** [review]

File: ofl/iansui/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  filename: "Iansui-Regular.ttf"
+  post_script_name: "Iansui-Regular"
+  full_name: "Iansui Regular"
+  copyright: "Copyright 2022 The Iansui Project Authors (https://github.com/ButTaiwan/iansui)."
+}
+subsets: "chinese-hongkong"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "symbols2"
```
Comment (chrissimpkins) [inline ofl/iansui/METADATA.pb:19]: @emmamarichal subset `symbols2` defined here

**Comment by emmamarichal** [comment]
@aaronbell @chrissimpkins 

having tofu in dev-sandbox:

Screenshot 2024-12-06 at 11 14 18

**Comment by aaronbell** [comment]
@emmamarichal what is the string used for that?

**Comment by emmamarichal** [comment]
鑑於對人權嘅無視同侮蔑已經發展為野蠻暴行，呢啲暴行玷污咗人類嘅良心，而一個人人享有言論同信仰自由並免予恐懼同匱乏嘅世界嘅來臨

**Pull request merged**